interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/out/FaultOutboundInterceptorTest.java

```diff
@@ -62,12 +33,21 @@ class FaultOutboundInterceptorTest extends AbstractSdtUnitTestBase {
     @Mock
     ServiceRequestDao mockServiceRequestDao;
 
+    RequestDaoService requestDaoService;
```

Comment:
Could this be a mock object rather than an actual object?

Reply:
Previously was the mock object and did not work. Kiran changed it to the actual to get it working